webpack.config.js

```diff
@@ -35,7 +35,7 @@ Encore
     // will require an extra script tag for runtime.js
     // but, you probably want this, unless you're building a single-page app
     //.enableSingleRuntimeChunk()
-    
```

Comment:
?

Reply:
soft tab got removed. My editor cleans this stuff up automatically.